aide_design/cdc_functions.py

```diff
@@ -91,28 +103,30 @@ def _flow_available(Diam, HeadlossCDC):
 
 # Length of tube required to get desired head loss at maximum flow based on 
 # the Hagen-Poiseuille equation.    
-@u.wraps(u.m, [u.m**3/u.s, u.m, u.m, u.m**2/u.s, None], False)
-def _len_tube(Flow, Diam, HeadLoss, Nu, KMinor):
-    num1 = pc.gravity.magnitude * HeadLoss * math.pi * (Diam**4)
-    denom1 = 128 * Nu * Flow
+@u.wraps(u.m, [u.m**3/u.s, u.m, u.m, u.kg/u.m**3, u.degK, None, None], False)
+def _len_tube(Flow, Diam, HeadLoss, conc_chem, temp, en_chem, KMinor):
```

Comment:
Unclear function name... perhaps this should be _len_tube_for_max_headloss or something suggesting that this is aiming for a certain headloss.